MH4U Dex/AreaDropTableViewCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AreaDropTableViewCell.h
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/11/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import "AreaDrop.h"
```

Comment:
It's usually preferable to forward-declare the class in the header file (`@class AreaDrop;`), rather than importing that class's header, unless there's something more than the class symbol that's necessary for this header.